cryostatio/cryostat-web
# feat(themes): add 'dark mode' theme

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
_______________________________________________

Fixes: #918

## Description of the change:
* Adds dark mode option to Cryostat. :smile: 
There may still be potential dark mode conflicting css styles in the code, I tried my best to fix them all.

* fixed popovers + tooltips to appendTo portalRoot instead of document body.
* renamed `Language & Region` Settings category to `General`.
* fixed dependency cycles with `Settings.tsx`

## How to manually test:
1. Run normally and go to settings and go to `General` to try out dark mode.
2. Try rendering different Cryostat pages and components and notice if there are any contrast/accesibility issues with the coloring scheme and add a review comment if so.
